vasyl_varyshchuk/Week_2/task_2/func_in_separate_processes.py

```diff
@@ -0,0 +1,88 @@
+""""
+This module returns random size of file name from list by processing elements in separate processes
+"""
+
+import random
+import string
+import ConfigParser
+from multiprocessing import Process
+
+config = ConfigParser.ConfigParser()
+path_to_config_file = '/home/vasyl/GL_Trainee/training_newcomers/vasyl_varyshchuk/Week_2/task_1/config.ini'
```

Comment:
It's a bad idea to have hardcoded path to file.

It's better to use configuration files, etc.

Reply:
I added separate configuration file according to your recommendation. This change is presented in last commit.